ui/apps/dotcms-block-editor/project.json

```diff
@@ -39,9 +39,9 @@
                         }
                     ],
                     "optimization": true,
-                    "outputHashing": "all",
+                    "outputHashing": "none",
```

Comment:
We sure? we might want this for cache no?

Reply:
In dotcms-block-editor we merge all to one file, and we need the same name always `cat dist/apps/dotcms-block-editor/{generator-runtime,runtime,polyfills,main}.js > dotcms-block-editor.js`